test/functional/http/post/common.js

```diff
@@ -81,41 +81,53 @@ function invalidAssets(option, badTransactions) {
 		}
 	});
 
-	describe('using invalid asset values', () => {
-		typesRepresentatives.allTypes.forEach(test => {
-			it(`using ${test.description} should fail`, () => {
-				transaction.asset = test.input;
+	describe('Using invalid asset values', () => {
```

Comment:
We generally use lowercase in descriptions.

Reply:
I think across the Lisk tests we are using root `description` with uppercase, and Mocha also suggest uppercase for the root `description` https://mochajs.org/#getting-started.

Reply:
If that's what we're following elsewhere then great. I think the Mocha examples in the link is because it's testing `Array` (which is spelled with a capital) rather than because it's a root-level test. Elsewhere in the docs they seem to use lowercase even in the root level, e.g. https://mochajs.org/#arrow-functions

Reply:
I think the Root description depends if we are testing a `class` and its method then it should be uppercase and lowercase for `methods` and other scenarios. here are two such cases wherein I can see the usage of uppercase 
https://mochajs.org/#asynchronous-code
https://mochajs.org/#asynchronous-hooks
Now the question is what should we follow going forward?